lox_services/utils/decorators.py

```diff
@@ -18,6 +18,25 @@
 )
 
 
+def production_environment_only(function: Callable):
+    """Decorator to ensure that a function is only ran in production environment.
+
+    Args:
+        func (function): Function to be decorated.
+    """
+
+    def wrapper(*args, **kwargs):
+        if get_env_variable("ENVIRONMENT") != "production":
+            print("Claiming script was ran non-production. This should not happen.")
+            raise RuntimeError(
+                "Claiming script was ran non-production. This should not happen."
+            )
+        else:
+            return function(*args, **kwargs)
+
+    return wrapper
+
+
```

Comment:
rename the decorator according to your exception message or vice versa.
with a decorator production_environment_only, I'm expecting to find it on function which are not linked to Claiming, but your error message saying it's claim related.